<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KK1SMTI" failures="2" tests="7" name="mytest3" time="54.911" errors="0" timestamp="2024-06-30T21:38:32 IST">
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="2.384"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="2.148"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="1.714">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDataDrivenTest.verifytest(TC_003_LoginDataDrivenTest.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verifytest -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.093"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="2.602"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.281">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDataDrivenTest.verifytest(TC_003_LoginDataDrivenTest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verifytest -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.251"/>
</testsuite> <!-- mytest3 -->
